debian/control

```diff
@@ -11,7 +11,7 @@ Vcs-Browser: https://github.com/rickysarraf/apt-offline
 
 Package: apt-offline
 Architecture: all
-Depends: ${misc:Depends}, ${python3:Depends}, apt, less, python3-magic
+Depends: ${misc:Depends}, ${python3:Depends}, apt, less, python3-magic, python3-apt
```

Comment:
This should go into Recommends. Because **python apt module** is not a hard dependency and I'd like to keep it this way.

Reply:
Heh!! Looks like this got in during the big commit chunk: 9fb140e01fe1d2facf0b97f923c0f4d34bebeae9

Reply:
Yes. So please just add it to Recommends for this PR.
Clean-up fix is needed in the master branch for this one, which I'll take care of.
https://github.com/rickysarraf/apt-offline/blob/08e9d20cd81e3a98304b2faecb3bcad544ee5ae5/apt_offline_core/AptOfflineCoreLib.py#L82

Reply:
> Yes. So please just add it to Recommends for this PR.
> Clean-up fix is needed in the master branch for this one, which I'll take care of.

Since the messy `import apt` line gonna be removed, there is no sence in my pull request.
Please, be sure to push the updated package to debian 10 backports.